pkg/utils/validation/cidr/cidr.go

```diff
@@ -41,6 +41,10 @@ type CIDR interface {
 	LastIPInRange() net.IP
 	// ValidateOverlap returns errors if the subnets do not overlap with CIDR.
 	ValidateOverlap(subsets ...CIDR) field.ErrorList
+	// ValidateMaxSize returns errors if the subnet is larger than the given bits. e.g. /15 is larger than /16
+	ValidateMaxSize(bits int) field.ErrorList
+	// ValidateMinSize returns errors if the subnet is smaller than the given bits. e.g. /16 is smaller than /15
+	ValidateMinSize(bits int) field.ErrorList
```

Comment:
I do not see that we use this function. Hence, please remove it. We can add it once we need it.

Reply:
I added it for completeness but fair enough.

Reply:
function removed